assets/javascripts/discourse/initializers/discourse-calendar.js

```diff
@@ -20,7 +20,10 @@ function loadFullCalendar() {
   );
 }
 
+let calendarPost;
+let calendar;
```

Comment:
I see a lot of removed `calendar` references. Do we still need to declare this here?

Reply:
Yes, since I had to access the `calendar` object from a function defined within its creation, I had to make the variable global. That's why I removed many of the references when passing it as an attribute between functions, but we still need it